### DISCUSSION TITLE
Full raw request + response (akin to the .http file client)

### DISCUSSION THREAD

**Discussion by vtali** [JetClient/jet-client-support#117] [General]
Do we have visibility what the JetClient is sending and receiving in raw format

**Comment by AntonShuvaev** [comment]
Yes, besides the toolwindow with the request collection, there’s also a toolwindow with the request preview (along with docs and code generation) and a console toolwindow displaying the requests and responses that were sent.

**Reply by vtali** [reply]
Thank you! Finally found the option, this is great. I appreciate the help!
